pkg/security/tests/accept_test.go

```diff
@@ -0,0 +1,126 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux && functionaltests
+
+// Package tests holds tests related files
+package tests
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+	"github.com/DataDog/datadog-agent/pkg/security/secl/rules"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/nettest"
+	"golang.org/x/sys/unix"
+	"math/rand/v2"
+	"net"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestAcceptEvent(t *testing.T) {
+
+	ruleDefs := []*rules.RuleDefinition{
+		{
+			ID:         "test_accept_af_inet",
+			Expression: `accept.addr.family == AF_INET && process.file.name == "syscall_tester"`,
+		},
+		{
+			ID:         "test_accept_af_inet6",
+			Expression: `accept.addr.family == AF_INET6 && process.file.name == "syscall_tester"`,
+		},
+	}
+
+	test, err := newTestModule(t, nil, ruleDefs)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer test.Close()
+
+	syscallTester, err := loadSyscallTester(t, test, "syscall_tester")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	const MIX = 4000
+	const MAX = 5000
+
+	t.Run("accept-af-inet-any-tcp-success", func(t *testing.T) {
+		port := rand.IntN(MAX-MIX) + MIX
+		go func() {
+			err := connectTo("AF_INET", "127.0.0.1", strconv.Itoa(port))
+			if err != nil {
+				t.Error(err)
+			}
+		}()
+		test.WaitSignal(t, func() error {
+			return runSyscallTesterFunc(context.Background(), t, syscallTester, "accept", "AF_INET", "0.0.0.0", strconv.Itoa(port))
+		}, func(event *model.Event, _ *rules.Rule) {
+			assert.Equal(t, "accept", event.GetType(), "wrong event type")
```

Comment:
you could use `assertTriggeredRule`  as well

Reply:
Done